pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -205,6 +205,13 @@ func (db *DB) Migration() *migrate.Migration {
 					`CREATE INDEX IF NOT EXISTS idx_bwa_serial ON bandwidth_agreements (serial_num)`,
 				},
 			},
+			{
+				Description: "Initiate Network reset",
+				Version:     4,
+				Action: migrate.SQL{
+					`UPDATE ttl SET expires = '1553641200' WHERE created <= '1553641200' `,
```

Comment:
can we have created be something in april? expires is fine. also aren't these numbers and not strings? we shouldn't have the quotes, right?

Reply:
I was expecting of SQL handling the cast, but i will double check 👍